src/libcommon/comm.h

```diff
@@ -123,6 +124,172 @@ enum class RequestNum {
     RECONSIDER_SKIPPED_UPDATE,
     UTILITY_QUIT,
 };
+inline std::string toString(RequestNum e) {
+    switch (e) {
+        case RequestNum::LOGIN_REQUESTTOKEN:
```

Comment:
Is there a Jira task for changing the case of enum values like `LOGIN_REQUESTTOKEN`?

Reply:
There wasn't, just created it -> https://infomaniak.atlassian.net/browse/KDESKTOP-1166